### DISCUSSION TITLE
Seeking Help with Git Issues: git status exited with code 128  //  fatal: cannot chdir to...

### DISCUSSION THREAD

**Discussion by RionNg** [Cyfrin/foundry-full-course-cu#1168] [Q&A]
Hello community,

<img width="778" alt="git status exited with code 128" src="https://github.com/Cyfrin/foundry-full-course-f23/assets/147371006/a7fccebe-ea78-4bac-a986-9525e48865f1">

I am currently facing some challenges with Git and would greatly appreciate your assistance in resolving them.
Below are the details of the issues:

.
**1. git status exited with code 128**
- Everything was fine.
- Then I proceed to Lesson 7 (In VS Code):
     1. mkdir foundry-fund-me-f23
     2. cd foundry-fund-me
     3. forge init
- Then I got this error "git status exited with code 128  //  fatal: cannot chdir to..."

.
**2. fatal: cannot chdir to...**
- I manually created a new folder "Cyfrin", then move the folder "foundry-f23" over (to look better).

.
I would be grateful for any insights, suggestions, or guidance you could provide to help resolve these Git issues.
Thank you in advance for your time and assistance!!

**Comment by xyz899** [comment]
### Hi, 

I think the problem comes from when you moved the project to another file since git was originally linked to its repo by the previous path of the folder, meaning Git hasn't been initalized to where you moved foundry-f23. try using `git status` `forge init --force` and if that does not work check how your git repo is managing the repo permissions. 

if anything does not work just check your saved files on github delete and git clone your project

**Reply by RionNg** [reply]
Thank you so much @Winter9998 for answering. 

Much appreciated!! ^_^
